recipes/iproute2/iproute2_4.8.0.oe

```diff
@@ -0,0 +1,11 @@
+require iproute2.inc
+
+SRC_URI="https://www.kernel.org/pub/linux/utils/net/${PN}/${PN}-${PV}.tar.xz"
+
+S="${SRCDIR}/${PN}-${PV}"
+
+DEPENDS += "iptables-dev"
+
+FILES_${PN}-tc += "${libdir}/tc/*"
+FILES_${PN}-dev += "${libdir}/tc/.debug/* \
+                    ${datadir}/bash-completion/completions/tc"
```

Comment:
bash-completion should not go in -dev package, since they are for tc either add them to the tc package or create a bash-completion